src/sw-template.js

```diff
@@ -0,0 +1,97 @@
+if ('function' === typeof importScripts) {
+    importScripts(
```

Comment:
I'm guessing we're loading the script here as opposed to `index.html` because this runs before index.html gets loaded?

Reply:
Great question.  My original assumption was that this was needed in sw-template.js for the build process when the service request is dynamically created.  

However, I now realize that none of the caching rules seem to require this.  Furthermore, I also observe that after commenting out the importScripts(), the build still succeeds and adds it again to the dynamically created service worker (found in build/sw.js).

At this point, I would need to read more into Workbox Build (dev documentation: https://developers.google.com/web/tools/workbox/modules/workbox-build) to understand some of the details of this code.

Reply:
Yeah I need to look into this as well! Thanks for sharing that link